release_dashboard/lib/widgets/progression.dart

```diff
@@ -133,6 +137,12 @@ class MainProgressionState extends State<MainProgression> {
                   isActive: true,
                   state: handleStepState(4),
                 ),
+                Step(
+                  title: Text(MainProgression._stepTitles[5]),
+                  content: PublishTagSubsteps(nextStep: nextStep),
+                  isActive: true,
```

Comment:
should these all be active at once?

Reply:
I changed to only when substep is completed or in progress, that substep is active. Otherwise, it is not.

Reply:
This file will be refactored with loops when handling this issue: https://github.com/flutter/flutter/projects/218

Reply:
Which issue (you linked the entire project)?